docs/tutorial.rst

```diff
@@ -394,3 +399,176 @@ estimates were obtained when Asian samples were compared with themselves.
+
+We begin by importing ``stdpopsim`` into a Python environment and specifying our desired
+species, *Arabidopsis thaliana*. From the :ref:`Catalog <sec_catalog>`, we can see that this
+specis has the ID ``AraTha``:
+
+.. code-block:: python
+
+    >>> import stdpopsim
+    >>> species = stdpopsim.get_species("AraTha")
+
+After skimming the :ref:`Catalog <sec_catalog>` to see our options, we'll specify our
+desired chromosome ``chr4`` and recombination map ``SalomeAveraged_TAIR7``.
+
+.. code-block:: python
+
+    >>> contig = species.get_contig("chr4", genetic_map="SalomeAveraged_TAIR7")
+
+
+From the API description, we can see that the :meth:`stdpopsim.IsolationWithMigration`
+model allows us to sample from a pair of populations that diverged from a common
+ancestral population. We'll specify that the effective population size of the ancestral
+population was 5000, that the population sizes of the two modern populations are 4000
+and 1000, that the populations diverged 1000 generations ago,
+and that rates of migration between the populations are 0.
+
+.. code-block:: python
+
+    >>> model = stdpopsim.IsolationWithMigration(NA=5000, N1=4000, N2=1000, T=1000, M12=0, M21=0)
+
+We'll simulate 10 chromosomes from each of the populations using the default
+simulation engine, which at the time of writing is ``msprime``.
+
+.. code-block:: python
+
+    >>> samples = model.get_samples(10,10)
+    >>> engine = stdpopsim.get_default_engine()
+
+Finally, we'll run a simulation using the objects we've created and store the outputted
+dataset in an object called ``ts``. For the purposes of this tutorial, we'll also run this
+simulation using a random seed:
+
+.. code-block:: python
+
+    >>> ts = engine.simulate(model, contig, samples, seed=13)
+
+--------------------------
+2. Calculating the SFS
+--------------------------
+
+Recall that the *site frequency spectrum* (SFS) summarises the distribution of allele
+frequencies in a given sample.
+At each site, there is an ancestral and (sometimes more than one) derived allele,
+and each allele is observed in the sample with some frequency.
+Each entry in the SFS corresponds to a particular sample frequency,
+and records the total number of derived alleles with that frequency.
+We can calculate the SFS directly from our tree sequence using the
+:meth:`tskit.TreeSequence.allele_frequency_spectrum` method.
+
+Since we wish to estimate the SFS separately for each of our two populations, we will
+first need to obtain subsets of the data corresponding to each set of samples. The
+:meth:`tskit.TreeSequence.samples`
+method in tskit allows us to find the IDs of samples from each population:
+
+.. code-block:: python
+
+    >>> pop0_samples = ts.samples(0)
+    >>> pop1_samples = ts.samples(1)
+    >>> print(pop0_samples)
+    >>> print(pop1_samples)
+    [0 1 2 3 4 5 6 7 8 9]
+    [10 11 12 13 14 15 16 17 18 19]
+
+We can then use :meth:`tskit.TreeSequence.simplify` to get the tree sequences
```

Comment:
Looks like @petrelharp comment of not using `simplify` hasn't been resolved yet.